packages/jest-circus/src/index.ts

```diff
@@ -54,6 +54,7 @@ const _dispatchDescribe = (
     asyncError.message = `Invalid second argument, ${blockFn}. It must be a callback function.`;
     throw asyncError;
   }
+  blockName = convertDescriptorToString(blockName);
```

Comment:
hmm, why is this needed now? Does that mean we didn't have runtime support in circus or jest-each? In that case  I'd like to see some runtime tests added 😅

Reply:
Ha, yeah, I think so!